app/plugins/Hosting/js/hosting.js

```diff
@@ -91,7 +91,7 @@ function update() {
 }
 
 // Called upon showing
-function show() {
+function start() {
```

Comment:
I thought it was more semantic while differentiating from the UI-level `plugin.show()` that also adds a transition fade and stuff